### PR TITLE
Added sales made per day bar chat

### PR DESCRIPTION
**Pull request by harmlessprince**

### CODE REVIEW

File: src/views/Admin/ReportingTool.vue [hunk 1]
```diff
@@ -546,6 +573,90 @@ export default {
...
+        let weekday = date.toLocaleString("en-us", {
+          weekday: dateConfig.weekday,
+        });
+        let year = date.toLocaleString("en-us", { year: dateConfig.year });
+        let day = date.toLocaleString("en-us", { day: dateConfig.day });
+        return [weekday, month, year].join("-");
+      }) ?? null;
+    },
+    getOrderValues() {
+      return this.reports.meta.totalSalesPerDay.map((item) => {
```
Comment (AltaraNg) [inline src/views/Admin/ReportingTool.vue:656]: rewrite line 656 `return this.reports?.meta?.totalSalesPerDay?.map(item =>  item.total);`

File: src/views/Admin/ReportingTool.vue [hunk 2]
```diff
@@ -546,6 +573,90 @@ export default {
...
+      };
+    },
+    getOrderDates() {
+      const dateConfig = {
+        weekday: "short",
+        year: "2-digit",
+        month: "short",
+        day: "numeric",
+      };
+      return this.reports.meta.totalSalesPerDay.map((item) => {
```
Comment (AltaraNg) [inline src/views/Admin/ReportingTool.vue:644]: Use optional chaining and remove ` ?? null`

File: src/views/Admin/ReportingTool.vue [hunk 3]
```diff
@@ -200,6 +200,24 @@
         </div>
       </div>
     </div>
+    <div class="container-fluid" v-show="productPieData">
+      <div class="card">
+        <h3 class="mx-5 my-5">Order By Day Statistics</h3>
+        <div class="card-body">
+          <div class="row">
+            <div class="col-md-12">
+              <template v-if="reports !== null">
```
Comment (AltaraNg) [inline src/views/Admin/ReportingTool.vue:209]: why not  `v-if="!reports"`

Comment (harmlessprince) [reply src/views/Admin/ReportingTool.vue:209]: isn't this for negating values that are boolean ?

Comment (AltaraNg) [reply src/views/Admin/ReportingTool.vue:209]: Not only booleans

**Review by AltaraNg — APPROVED** [review]

**Pull request merged**